pkg/cmd/init/init.go

```diff
@@ -0,0 +1,23 @@
+package init
+
+import "github.com/spf13/cobra"
+
+func NewCmdInit() *cobra.Command {
```

Comment:
Why did you use the prefix New for the function?

Reply:
I have decided to use more or less the style used by the gh cli as you can see

https://github.com/cli/cli/blob/6783509d3bfdf55e1007dbedf34f2bdcd9fd1ec1/pkg/cmd/root/root.go#L39